src/components/course-list/Course.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import Button from '@material-ui/core/Button'
+import DeleteIcon from '@material-ui/icons/Delete';
+import EditIcon from '@material-ui/icons/Edit';
+import Typography from '@material-ui/core/Typography'
+import './Course.css';
+
+const styles = theme => ({
+    button: {
+        margin: theme.spacing.unit,
+    },
+    extendedIcon: {
+        marginRight: theme.spacing.unit,
+    },
+});
+
+
+class Course extends React.Component {
```

Comment:
why folder named `course-list` seems like mistake in folder naming

Reply:
fixed